airflow/dag_processing/bundles/base.py

```diff
@@ -48,6 +260,10 @@ class BaseDagBundle(ABC):
     """
 
     supports_versioning: bool = False
+
+    bundle_type: str
+    """This is used in generating some the paths used for local storage."""
```

Comment:
```suggestion
    """This is used in generating some paths used for local storage."""
```

Might be good to have an example...